docs/usage/examples/opentelemetry.md

```diff
@@ -72,12 +93,23 @@ service:
   pipelines:
     traces:
       receivers: [otlp]
-      exporters: [otlp/jaeger, logging]
-      processors: [spanmetrics, batch]
+      exporters:
+        - otlp/jaeger
+        # Send traces to connector for metrics calculation
+        - spanmetrics
+        # Enable debug exporter to see traces in the logs
+        #- debug
```

Comment:
Here's another commented out bit: `- debug`.

Reply:
For user debugging